pkgs/applications/networking/juju/default.nix

```
@@ -0,0 +1,25 @@
+{ lib, pkgs }:
+
+pkgs.buildGoModule rec {
```

```suggestion
{ lib, buildGoModule }:

buildGoModule rec {
```

Using pkgs as an input is not allowed.

---

Fixed.